### PR TITLE
Clarify PatchVertices documentation for tessellation shaders.

### PR DESCRIPTION
**Pull request by Venemo**
I attempted to make the `PatchVertices` documentation clearer. See #1461 for the discussion that motivated this change.

### CODE REVIEW

**Comment by Venemo** [comment]
I already signed the CLA but it seems that it's still "Pending" here...

**Comment by oddhack** [comment]
@outofcontrol could you take a look at the CLAbot for this PR? It's saying "Timur Kristóf seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please add the email address used for this commit to your account."

@Venemo in any event, it usually takes us a week or two to get the Vulkan WG to sign off on PRs, so this is unlikely to hold things up any further.

**Comment by Khronoswebmaster** [comment]
@Venemo Is it possible you committed this PR with a different email address from the one used on your GitHub account? This is the most common issue we see for this error. More [here](https://docs.github.com/en/github/committing-changes-to-your-project/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user).

File: chapters/interfaces.txt
Comment (gnl21) [inline chapters/interfaces.txt:2757]:
```suggestion
In a Tessellation Evaluation Shader, code:PatchVertices is equal to the
```

Just a minor quibble, but I'd prefer to say "equal" here to emphasize that the value is the same but they're not "equivalent" because they have different uses. You can't substitute one for the other.

File: chapters/interfaces.txt
Comment (gnl21) [inline chapters/interfaces.txt:2759]:
```suggestion
tessellation control output patch size.
```

I don't think it really helps with understanding here to say how these things are specified, and the more places we say things like that, the more places we have to keep consistent, etc, so I'd rather leave this last clause out.

File: chapters/interfaces.txt
Comment (gnl21) [inline chapters/interfaces.txt:2764]: This seems a little bit repetitive now. Maybe it would be sufficient to say:
> The same tessellation control or tessellation evaluation shader may be used in different pipelines where the patches are of differing sizes, so the value of the code:PatchVertices variable may differ.

(Github won't let me include that as a suggestion because it spans a deleted line.)

**Review by gnl21 — COMMENTED** [review]
Everything that's here looks correct to me, and I'd be OK taking this as it is. I've made a few editorial suggestions that I think would improve it. What do you think, @Venemo?

File: chapters/interfaces.txt
Comment (Venemo) [reply chapters/interfaces.txt:2757]: Shouldn't we then just say `code::PatchVertices is the same as the tessellation control output patch size`?

File: chapters/interfaces.txt
Comment (Venemo) [reply chapters/interfaces.txt:2759]: Ok, let's leave it out.

File: chapters/interfaces.txt
Comment (Venemo) [reply chapters/interfaces.txt:2764]: Agreed, let's get rid of the repetitiveness. I think this might be even more concise:

> When the same shader is used in different pipelines where the patch sizes are configured differently, the value of the code:PatchVertices variable will also differ.

How does this sound?

File: chapters/interfaces.txt
Comment (Venemo) [reply chapters/interfaces.txt:2757]: On a second thought, I think "is equal to" sounds better.

Comment (gnl21) [reply chapters/interfaces.txt:2757]: They're not the same thing, but they have the same value, so I'm not sure saying that they are "the same" is clearer. I think saying that they're equal probably conveys that best.

Comment (Venemo) [reply chapters/interfaces.txt]: Agreed.

File: chapters/interfaces.txt
Comment (gnl21) [reply chapters/interfaces.txt]: Sounds good to me. Thanks.

Comment (Venemo) [reply chapters/interfaces.txt]: Done.

**Comment by Venemo** [comment]
@Khronoswebmaster Thanks, that was exactly the problem. I forgot that I use a different email address for my GitHub login.

**Review by gnl21 — APPROVED** [review]

**Comment by Khronoswebmaster** [comment]
> @Khronoswebmaster Thanks, that was exactly the problem. I forgot that I use a different email address for my GitHub login.

Happy to see it was a simple fix.

**Comment by Venemo** [comment]
@gnl21 What is the process of merging this commit? I see that you approved it but I don't have a merge button here.

**Comment by gnl21** [comment]
@oddhack will merge this when he's ready to make a spec update with it in. (Because a lot of the content of this repo gets brought across from Khronos's internal repo, it's easier to handle it this way). If I reviewed it in time then that will probably be sometime around this weekend.

**Comment by Venemo** [comment]
Awesome, thank you!

**Review by SidBlack — COMMENTED** [review]
ok

**Pull request merged**
